qiskit_nature/problems/second_quantization/molecular/integrals_calculators/angular_momentum_integrals_calculator.py

```diff
@@ -0,0 +1,114 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Calculator of 1- and 2-body integrals for a total angular momentum."""
+
+import itertools
+from typing import Tuple
+
+import numpy as np
+
+
+def calc_total_ang_momentum_ints(num_modes: int) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
Are we going to directly expose function via the init files - I see a lot of empty init files (just copyright I mean)